docs/anonymity-configuration.rst

```diff
@@ -184,44 +184,50 @@ Anonymity configuration
 =======================
 
 Tahoe-LAFS provides a configuration "safety flag" for explicitly stating
-whether or not anonymity is required for a node::
+whether or not IP-address privacy is required for a node::
 
```

Comment:
I've opened https://trac.i2p2.de/ticket/1842 for adding a tl;dr to the I2P website.